reliabletopic/src/main/java/com/hazelcast/jet/contrib/reliabletopic/ReliableTopicSource.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.contrib.reliabletopic;
+
+import com.hazelcast.function.FunctionEx;
+import com.hazelcast.jet.core.Processor.Context;
+import com.hazelcast.jet.pipeline.SourceBuilder;
+import com.hazelcast.jet.pipeline.StreamSource;
+import com.hazelcast.topic.Message;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayDeque;
+
+import java.util.Queue;
+
+
+
+/**
+ * Contains a method for creating a reliable topic stream sources.
+ */
+public final class ReliableTopicSource {
+
+    private ReliableTopicSource() {
+    }
+
+    /**
+     * Creates a {@link StreamSource} which reads messages from
+     * from a Hazelcast reliable topic with the specified name.
+     * @param topic       topic name to read from
+     * @param projectionFn built in mapping function of the source which can be
+     *                     used to map {@link Message} instances received
+     *                     from a reliable topic to an arbitrary type of output;
+     *                     it's included for convenience.
+     * @param <M> type of item inside a message.
+     * @param <T> type of emitted item after projection.
+     * @return {@link StreamSource}
+     */
+    public static <M, T> StreamSource<T> topicSource(
```

Comment:
`HazelcastInstance` has 2 methods:
- `ITopic getTopic()`
- `ITopic getReliableTopic()`

As far as I understand these don't share the namespace, but have the same interface. Because of this maybe we should support both (or we do already support the other?). The javadoc doesn't specify any difference between them... But we should be clear which one we mean and use the terminology. So this method should probably be named `reliableTopicSource`.